app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for relationship, remote: true do |f| %>
+  <div><%= hidden_field_tag :followed_id, @user.id %></div>
```

Comment:
`<%= f.hidden_field :followed_id, @user.id %>`

Reply:
lỗi chệ ơi, :((
undefined method `merge' for 3:Integer

Reply:
Check lại đi nhé :D